blackbox/docs/general/builtins/aggregation.rst

```diff
@@ -31,13 +31,48 @@ query result into one row, with one returned aggregation value.
 For a tabulated summary of aggregation functions, see
 :ref:`sql_dql_aggregation`.
 
+.. _aggregate-expressions:
+
+Aggregate Expressions
+=====================
+
+An aggregate expression represents the application of an aggregate function
```

Comment:
This partially repeats the introduction section (where we're referring only to `aggregation function`). Maybe we could re-work the introduction as well, so that the flow of both sections here feels a bit more natural and fits better together?